packages/block-library/src/post/edit/placeholder.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityBlockEditor, EntityProvider } from '@wordpress/core-data';
+import { __ } from '@wordpress/i18n';
+import { BlockPreview } from '@wordpress/block-editor';
+import { useState, useCallback } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { Placeholder, TextControl, Button } from '@wordpress/components';
+
+function PostPreview() {
+	const [ blocks ] = useEntityBlockEditor( 'postType', 'post' );
+	return (
+		<div className="wp-block-post__placeholder-preview">
```

Comment:
Two accessibility-related questions:

1. Is this a preview similar to a full post preview one can invoke from the top toolbar? If so, this would need a role="document" so screen readers that invoke a virtual reading mode can be sure to get this even if the surrounding controls would normally cause it to be in focus/direct interaction mode.
2. Should this be tabable via the keyboard? If so, it would also need tabindex="0" or a similar mechanism to make it focusable so keyboard users can reach it.

Reply:
1. No.
2. No.

It's a small preview of the post with the input's current ID.